designsafe/static/scripts/data-depot/components/published/published.component.js

```diff
@@ -48,6 +47,15 @@ export class PublishedDataCtrl {
                         baseUrl: '/api/public/files'
                     };
 
+                    this.getFileObjs = (evt) => {
```

Comment:
Can you define this as a class method and outside this callback?

Reply:
Sounds good I'll change it.